framework/include/geomsearch/FindContactPoint.h

```diff
@@ -26,8 +26,9 @@ namespace Moose
 
 void
 findContactPoint(PenetrationInfo & p_info,
-                 FEBase * _fe,
-                 FEType & _fe_type,
+                 FEBase * fe_elem,
+                 FEBase * fe_side,
+                 FEType & fe_side_type,
```

Comment:
I see your point.  In this case, though, the name change kind of goes hand in hand with the fix. I'm passing in another FEBase object into this function, and I needed to make the distinction between the one associated with the element and the one associated with the side.  Since there was previously only one such object, it wasn't named in a way that indicated that it was associated with the side.